engines/wintermute/base/gfx/base_renderer.h

```diff
@@ -45,6 +45,18 @@ class BasePersistenceManager;
 class Camera3D;
 #endif
 
+struct FogParameters
+{
```

Comment:
format '{'

Reply:
Fixed and in face, the struct is in a proper place now.